code/tests/_game_test_puppeteer.dm

```diff
@@ -48,6 +48,16 @@
 
 	origin_test.Fail("could not spawn obj [obj_type] near [src]")
 
+/datum/test_puppeteer/proc/use_item_in_hand()
+	var/obj/item/item = puppet.get_active_hand()
+	if(!item)
+		origin_test.Fail("could not find obj in [puppet] active hand", __FILE__, __LINE__)
+		return
```

Comment:
this __FILE__ and __LINE__ will only ever print this line in this file, not the line in the origin test. you could have a return value here checked by the test which then uses a test macro that uses these